Website (#179)
tweag/monad-bayes: monad-bayes-site/about.md

**Define distributions** [as programs](../Introduction.html)

> **Perform inference** [with a variety of standard methods, which you can extend](/tutorials.html) 

Reviewer: ```suggestion
**Perform inference** [with a variety of standard methods](/tutorials.html) 
```
I liked previous formulation better - it was concise.

> **Integrate with arbitrary Haskell code** [like this](/examples.html) because Monad-Bayes is just a library, not a separate language

Reviewer: ```suggestion
**Integrate with Haskell code** [like this](/examples.html) because Monad-Bayes is just a library, not a separate language
```
I think those main advantages of the monad-bayes should be presented in a short form and elaborated later if needed.

<!-- ```haskell
example :: Distribution Histogram
example = fmap histogram (replicateM 1000 (normal 0 1))
``` -->

# Example

> <img src="../images/code_example.png" 

Reviewer: If we could use a regular code instead of a picture it would be better - a reader could copy it if needed and it the site will be more coherent.

Author: Good point, I agree. Some other feedback on the website was given here: https://discourse.haskell.org/t/probabilistic-programming-in-haskell/4867/4

Reviewer: Addressed in https://github.com/tweag/monad-bayes/pull/201

     width="720" 
     height="300" />

> `model` is a mixture of Gaussians, defined as a program. Its type `Distribution Double` shows that it is a distribution over reals. `image` is a program too: as its type shows, it is a distribution over plots. In particular, plots that arise from forming a 200 bin histogram out of 100000 iid draws from `model`. To sample from `image`, we simply write `sampler image`, with the result shown below:

Reviewer: ```suggestion
The program `model` is a mixture of Gaussians. Its type `Distribution Double` represents a distribution over reals. 
`image` is a program too: as its type shows, it is a distribution over plots. In particular, plots that arise from forming a 200 bin histogram out of 100000 independent identically distributed (iid) draws from `model`. 
To sample from `image`, we simply write `sampler image`, with the result shown below:
```
Please use new lines for new sentences - it will be easier to read.

Author: Minor edit:

"The program `model` is a mixture of Gaussians. Its type `Distribution Double` represents a distribution over reals. "

Reviewer: ok
